relationships.md

```diff
@@ -94,6 +94,23 @@ The 2 items are included as HAL links but are not embedded
 
 An optional leftwardValue and rightwardValue property can be present. It's omitted when it's null.
 
+## Search methods
+
+### Relationship involving specified items
+**/api/core/relationshiptypes/search/byItemsAndType?typeId=<:relationship-type-id>&leftItem=<item-uuid1>[&...&&leftItem=<item-uuidN>]&rightItem=<item-uuid1>[&...&&rightItem=<item-uuidN>]**
```

Comment:
This endpoint starts with `/api/core/relationshiptypes`.. I think it should be `/api/core/relationships`?

Reply:
fixed